### ISSUE TITLE
Prism workflow

### ISSUE THREAD

**Issue by LBarranco** [GettEngineering/Prism#36]
Hi, 
I was wondering to know if there is the possiblity to allow Prism to automatically download the updated colours/font/whatever directly from Zeplin even when the application is live. 

USE CASE:
1) I released on the store the myApp v:1.0.0 where all my button are blue
2) I change the buttonColor inside the Zeplin board from blue to green
3) The very next time I open myApp it automatically download the new colour file and magically all my buttons turns to green.

In my understanding in this moment this is not possible, because I have to manually generate the colours file from the console and then later add it in my Xcode project. 

Is it correct ?

Thank you very much!

**Comment by freak4pc** [comment]
Hey @LBarranco :) 

This is a pretty neat idea, even though I have some reservations about doing something like this in production, mainly if one of my designers randomly/accidentally changes a color it would effect my production environment with no review process (e.g. Zeplin is really easy to change, remove colors, etc).

That's why I personally opted for a CI solution since it still has a middle-step where the developer actively decided whether or not to merge the code (more on our specific flow here if you're interested: https://youtu.be/SI8rQp8HaQU?t=5520 (starts at 1hr32m)

Regarding your question:

Yes, Prism was not design as a CLI tool to that; but — Prism CLI sits on top a of a separate dependency - `PrismCore`. Nothing prevents you from embedding PrismCore in your app and invoke the needed APIs on startup to fetch the latest colors before your app starts. 

Might be a good topic for a video some time ;)

tl;dr - Yes, you can easily do it but it's not sometihng Prism CLI was planned for
